test/transport_web/solutions/reuse_data_test.exs

```diff
@@ -12,7 +12,7 @@ defmodule TransportWeb.ReuseDataTest do
   hound_session()
 
   @tag :integration
-  test "I can click somewhere to reuse transport data" do
+  test "I can click on the map to reuse transport data" do
```

Comment:
This test should fail now.

Reply:
Nop, we still need a page per organization to change this test

Reply:
Why ? Wouldn't it be enough to add a link to the .zip file ?

Reply:
I'm not a fan, if you're using the site for the first time you'll want info about the licence for example.

Reply:
OK